rpc_storage_handler.go

```diff
@@ -999,24 +999,67 @@ func (r *RPCStorageHandler) CheckForKeyspaceChanges(orgId string) {
 	}
 }
 
+func (r *RPCStorageHandler) getKeyForCreation(keyName string) (string, string, error) {
```

Comment:
We can do that since GetRawKey is just sitting there doing nothing at the moment - however this function is more specific in that it will always look for keys with the apikey prefix

Reply:
Lets then generalize it, we will for sure need `GetRawKey` in future.